src/timeline.ts

```diff
@@ -90,61 +100,64 @@ class Timeline implements IAppView {
       .attr('height', h);
 
     //width of the timeline div
-    var widthTimelineDiv = $('#timeline').width();
+    let widthTimelineDiv = $('#timeline').width();
 
-    svgtimeline.append('line')
-      .style('stroke', 'black')
-      .attr('x1', 0)
-      .attr('y1', 60)
-      .attr('x2', widthTimelineDiv-10)
-      .attr('y2', 60);
+      function scaleCircles(widthTimelineDiv) {
+      //Padding for the circles
+      const padding = 20;
+      //showing only 7 circles on the timeline when no time-object is availiable for the specific dataset
+      // in the next step -> implement the feature of a scroll bar showing more data points on the timeline
+      const numberofCircles = 7;
+      return (widthTimelineDiv - padding) / numberofCircles;
+    }
 
-    //helper variable for clicking event
-    var isClicked = 0;
+     //helper variable for clicking event
+    let isClicked = 0;
     //overall time span in days
-    var firstTimePoint = moment(items[0].time);
-    var lastTimePoint =  moment(items[items.length-1].time);
-    var timeRange = lastTimePoint.diff(firstTimePoint, 'days');
+    const firstTimePoint = moment(items[0].time);
+    const lastTimePoint = moment(items[items.length - 1].time);
+    const timeRange = lastTimePoint.diff(firstTimePoint, 'days');
 
     // Abbildungsbereich = Width
     // Skalierungfaktor = Width / Time Range
 
 
     const xScaleTime = d3.scale.linear()
       .domain([0, timeRange])
-      .range([20, widthTimelineDiv-20]); // 20 = Spacing
+      .range([20, widthTimelineDiv - 20]); // 20 = Spacing
 
-    function scaleCircles(widthTimelineDiv) {
-      //Padding for the circles
-      const padding = 20;
-      //showing only 7 circles on the timeline when no time-object is availiable for the specific dataset
-      // in the next step -> implement the feature of a scroll bar showing more data points on the timeline
-      const numberofCircles = 7;
-      return (widthTimelineDiv - padding) / numberofCircles;
-    }
+    drawTimeline();
 
+    function drawTimeline() {
```

Comment:
Check line indent of this function (and the whole source code). You can use Code -> Auto Indent Lines of PyCharm.